nebula_decoders/src/nebula_decoders_continental/decoders/continental_ars548_decoder.cpp

```diff
@@ -0,0 +1,574 @@
+      << std::to_string(sensor_status_packet.status.sensor_ip_address03);
+  radar_status_.sensor_ip_address0 = ss0.str();
+
+  ss1 << std::to_string(sensor_status_packet.status.sensor_ip_address10) << "."
+      << std::to_string(sensor_status_packet.status.sensor_ip_address11) << "."
+      << std::to_string(sensor_status_packet.status.sensor_ip_address12) << "."
+      << std::to_string(sensor_status_packet.status.sensor_ip_address13);
+  radar_status_.sensor_ip_address1 = ss1.str();
+
+  radar_status_.configuration_counter = sensor_status_packet.configuration_counter;
+
+  radar_status_.longitudinal_velocity_status =
+    sensor_status_packet.longitudinal_velocity_status == 0   ? "VDY_OK"
+    : sensor_status_packet.longitudinal_velocity_status == 1 ? "VDY_NOTOK"
+                                                             : "INVALID VALUE";
+
+  radar_status_.longitudinal_acceleration_status =
+    sensor_status_packet.longitudinal_acceleration_status == 0   ? "VDY_OK"
+    : sensor_status_packet.longitudinal_acceleration_status == 1 ? "VDY_NOTOK"
+                                                                 : "INVALID VALUE";
+
+  radar_status_.lateral_acceleration_status =
+    sensor_status_packet.lateral_acceleration_status == 0   ? "VDY_OK"
+    : sensor_status_packet.lateral_acceleration_status == 1 ? "VDY_NOTOK"
+                                                            : "INVALID VALUE";
+
+  radar_status_.yaw_rate_status = sensor_status_packet.yaw_rate_status == 0   ? "VDY_OK"
+                                  : sensor_status_packet.yaw_rate_status == 1 ? "VDY_NOTOK"
+                                                                              : "INVALID VALUE";
+
+  radar_status_.steering_angle_status = sensor_status_packet.steering_angle_status == 0 ? "VDY_OK"
+                                        : sensor_status_packet.steering_angle_status == 1
+                                          ? "VDY_NOTOK"
+                                          : "INVALID VALUE";
+
+  radar_status_.driving_direction_status =
+    sensor_status_packet.driving_direction_status == 0   ? "VDY_OK"
+    : sensor_status_packet.driving_direction_status == 1 ? "VDY_NOTOK"
+                                                         : "INVALID VALUE";
+
+  radar_status_.characteristic_speed_status =
+    sensor_status_packet.characteristic_speed_status == 0   ? "VDY_OK"
+    : sensor_status_packet.characteristic_speed_status == 1 ? "VDY_NOTOK"
+                                                            : "INVALID VALUE";
+
+  if (sensor_status_packet.radar_status == 0) {
+    radar_status_.radar_status = "STATE_INIT";
+  } else if (sensor_status_packet.radar_status == 1) {
+    radar_status_.radar_status = "STATE_OK";
+  } else if (sensor_status_packet.radar_status == 2) {
+    radar_status_.radar_status = "STATE_INVALID";
+  } else {
+    radar_status_.radar_status = "INVALID VALUE";
+  }
```

Comment:
These would be cleaner as switch statements

Reply:
Addressed in [800b351](https://github.com/tier4/nebula/pull/123/commits/800b351ad5c941b14e31507c4cd59eb6b42ddd4a)